addons/survey/tests/test_survey.py

```diff
@@ -29,36 +29,34 @@ def test_answer_validation_mandatory(self):
                 kwargs['labels_2'] = [{'value': 'Row0'}, {'value': 'Row1'}]
             question = self._add_question(self.page_0, 'Q0', question_type, **kwargs)
 
-            answer_tag = '%s_%s' % (self.survey.id, question.id)
             self.assertDictEqual(
-                question.validate_question({answer_tag: ''}, answer_tag),
-                {answer_tag: 'TestError'}
+                question.validate_question(''),
+                {question.id: 'TestError'}
             )
 
     @users('survey_manager')
     def test_answer_validation_date(self):
         question = self._add_question(
             self.page_0, 'Q0', 'date', validation_required=True,
             validation_min_date='2015-03-20', validation_max_date='2015-03-25', validation_error_msg='ValidationError')
-        answer_tag = '%s_%s' % (self.survey.id, question.id)
 
         self.assertEqual(
-            question.validate_question({answer_tag: 'Is Alfred an answer ?'}, answer_tag),
-            {answer_tag: _('This is not a date')}
+            question.validate_question('Is Alfred an answer ?'),
```

Comment:
`Is Alfred an answer ?` should definitely be changed to `La mer noire`

Reply:
J'aime ! :D